friends/src/components/EditFriend.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import axios from "axios";
+import { Button, Form, FormGroup, Label, Input } from "reactstrap";
+
+class EditFriend extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      age: "",
+      email: ""
+    };
+  }
+  //  Handler to capture the text input and assign it to state
+  handleChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  componentDidMount = () => {
```

Comment:
Very nice work prepopulating the form with the current values! If the fields aren't required by the server it might be good to set the state like:
```
this.setState({
          name: friend.name ? friend.name : "",
          age: friend.age ? friend.age : "",
          email: friend.email ? friend.email : ""
        });
```
I believe that if those values are undefined because they're null in the database then React might complain or crash. It's always good to consider and account for possible outliers in data you may receive.